src/pyj/read_book/view.pyj

```diff
@@ -827,10 +850,10 @@ class View:
         else:
             self.show_name(name, initial_position=pos)
         sd = get_session_data()
-        c = sd.get('controls_help_shown_count', 0)
+        c = sd.get('controls_help_shown_count' + ('_rtl_page_progression' if rtl_page_progression() else ''), 0)
```

Comment:
Since the tutorial overlay changes based on reading mode, I thought it made sense to show it again if a book is in a different reading mode, and just doing it twice for each mode seemed like the simplest solution.